docs/GettingStarted.rst

```diff
@@ -1184,8 +1184,39 @@ The article is shared into the space's activity stream.
 
 
 .. note:: The |image157| button is grey and unclickable until the mandatory fields *title* and *content* are filled. 
+
          
-		  
+Sharing a news from full form
+----------------------------------
+
+To share an article from the full form, you have to display the small form as explained in the previous paragraph and click on "More options" button.
```

Comment:
I would add a sentence to explain what is the full form and why the user would want to use it, something like "A more detailed form is available to post a news in case you want to add a summary and an illustration.".